src/components/LearnMoreSection/LearnMoreSection.css

```diff
@@ -0,0 +1,61 @@
+.learn-more-section {
+  display: flex;
+  align-items: center;
+  gap: 100px;
+}
+.learn-more-section .learn-more-image {
+  width: 660px;
+  height: 540px;
+}
+.learn-more-section .learn-more-content {
+  width: 540px;
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+}
```

Comment:
- [ ] Here instead of applying a width, make them equal sized using flex and then apply appropriate gap and paddings

Reply:
Fixed